tests/assets/ansible/roles/caasp/tasks/Suse.yaml

```diff
@@ -32,45 +32,38 @@
     group: root
     mode: 0644
 
-- name: install skuba & kubectl
+- name: install skuba / kubectl / helm
   zypper:
-      name:
-        - patterns-caasp-Management
+      name: ['patterns-caasp-Management', 'helm3']
       state: present
       update_cache: no
   when: inventory_hostname in groups['first_master']
 
-- name: retrieve skuba binary into workspace
+- name: retrieve skuba / kubectl / helm binaries into workspace
   fetch:
-    src: /usr/bin/skuba
+    src: /usr/bin/{{ item }}
     dest: '{{ rookcheck_workspace_dir }}/bin/'
     flat: yes
+  loop:
+    - skuba
+    - kubectl
+    - helm3
   when: inventory_hostname in groups['first_master']
 
-- name: retrieve kubectl binary into workspace
-  fetch:
-    src: /usr/bin/kubectl
-    dest: '{{ rookcheck_workspace_dir }}/bin/'
-    flat: yes
-  when: inventory_hostname in groups['first_master']
-
-- name: make skuba binary executable
-  file:
-    path: '{{ rookcheck_workspace_dir }}/bin/skuba'
-    mode: 0755
-  delegate_to: 127.0.0.1
-  when: inventory_hostname in groups['first_master']
-
-- name: make kubectl binary executable
+- name: make skuba / kubectl / helm binaries executable
   file:
-    path: '{{ rookcheck_workspace_dir }}/bin/kubectl'
+    path: '{{ rookcheck_workspace_dir }}/bin/{{ item }}'
     mode: 0755
   delegate_to: 127.0.0.1
+  loop:
+    - skuba
+    - kubectl
+    - helm3
   when: inventory_hostname in groups['first_master']
 
-- name: remove skuba
+- name: remove unecessary packages
   zypper:
-      name: skuba
+      name: ['patterns-caasp-Management', 'helm3']
```

Comment:
does that work for patterns? eg `sudo zypper rm laptop` does noting on my local machine, but `sudo zypper rm -t pattern laptop` would remove the pattern `laptop`.

Reply:
Looks like it works
```
rookcheck-bleon-3d5d-master-0:~ # zypper rm patterns-caasp-Management
Reading installed packages...
Resolving package dependencies...

The following package is going to be REMOVED:
  patterns-caasp-Management

The following pattern is going to be REMOVED:
  SUSE-CaaSP-Management

1 package to remove.
After the operation, 56.0 B will be freed.
Continue? [y/n/v/...? shows all options] (y): y
(1/1) Removing patterns-caasp-Management-4.5-1.3.x86_64 ........................................................................................................................................................................[done]
 
rookcheck-bleon-3d5d-master-0:~ #
```